docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -3,41 +3,70 @@ Datasets
 Check out page :doc:`Speech Classification Datasets <../speech_classification/datasets>` and :doc:`Speaker Recogniton Datasets <../speaker_recognition/datasets>` 
 for preparing datasets for training and validating VAD and speaker embedding models respectively.
 
-For Speaker Diarization inference, ``diarizer`` expects either list of paths to audio files or a file containing absolute paths to audio files. 
+For Speaker Diarization inference, ``diarizer`` expects following arguments, which needs to be created by user depending on dataset:
```

Comment:
expects following arguments, which needs to be created by user depending on dataset: -> expects the following arguments: